### PR TITLE
Rename package and files to qibo-client

### PR DESCRIPTION
**Pull request by marcorossi5**
This PR addresses #11

### CODE REVIEW

File: src/qibo_client/qibo_client.py [hunk 1]
```diff
@@ -255,3 +244,15 @@ def _get_result(self) -> Optional[np.ndarray]:
 
         self.results_path = self.results_folder / "results.npy"
         return qibo.result.load_result(self.results_path)
+
+
+def TII(token: str) -> Client:
```
Comment (scarrazza) [inline src/qibo_client/qibo_client.py:249]: Please move this to another file, e.g. tii.py and import the TII class in the init.

**Comment by scarrazza** [comment]
@marcorossi5 could you please fix conflicts and CI?

**Comment by marcorossi5** [comment]
@scarrazza, the coverage is 100% and the merge conflicts are fixed. If you are satisfied with the code we can merge.

**Review by scarrazza — APPROVED** [review]
Thank you.

**Pull request merged**